sormas-backend/src/main/java/de/symeda/sormas/backend/importexport/ImportFacadeEjb.java

```diff
@@ -124,6 +125,34 @@
 
 	private final Logger logger = LoggerFactory.getLogger(getClass());
 
+	private static final List<String> EVENT_COLUMNS_TO_REMOVE = Arrays.asList(EventDto.SUPERORDINATE_EVENT);
```

Comment:
I think it would be good to allow users to specify a superordinate event via the event import by providing its UUID if they know it; could you please implement the necessary changes?

Reply:
Indeed, I wanted to ask you information of this part after making the PR but forgot to do it...
By the way, thanks for reporting me this issue, it's done now.